repo-agent/repowatch/controllers/repowatch_controller.go

```diff
@@ -147,7 +147,49 @@ func (r *RepoWatchReconciler) reconcileReviews(ctx context.Context, repoWatch *r
 	log := log.FromContext(ctx)
 	log.Info("reconciling reviews")
 
+	explicitPRs := r.getExplicitPRs(ctx, ghClient, repoWatch, owner, repo)
+
+	prs, err := r.listOpenPRs(ctx, ghClient, owner, repo)
+	if err != nil {
+		return err
+	}
+
+	prs = r.filterPRsByLabels(prs, repoWatch)
+	prs = r.deduplicatePRs(prs, explicitPRs)
+	prs = r.sortPRs(ctx, ghClient, prs, repoWatch)
+
+	// Log repoIssues and sandboxList for debug purposes
+	prsStr := []string{}
+	for _, pr := range prs {
+		prsStr = append(prsStr, fmt.Sprintf("%d", *pr.Number))
+	}
+	log.Info("DEBUG INFO PRs:", "prs", prsStr)
```

Comment:
nit: should we use a different log level for debug info?

Reply:
done.